src/Nethermind/Nethermind.Consensus/Producers/PayloadAttributes.cs

```diff
@@ -27,6 +28,8 @@ public class PayloadAttributes
 
     public Deposit[]? Deposits { get; set; }
```

Comment:
@Demuirgos @rjnrohit are you sure that deposits and exists should be added to payloadAttributes? I don't think so